README.md

```diff
@@ -1,10 +1,24 @@
 # <img src="./files/images/thunder.svg?sanitize=true" alt="Thunder" width="4%"/> Thunder [![Tweet](https://img.shields.io/twitter/url/http/shields.io.svg?style=social)](https://twitter.com/intent/tweet?text=I%20just%20tried%20ThunderCMS.%20The%20CMS%20for%20professional%20publishing!&url=https://www.thunder.org&via=ThunderCoreTeam&hashtags=drupal8,publishing,ThunderCMS)
 
-[![Build Status](https://travis-ci.org/BurdaMagazinOrg/thunder-distribution.svg?branch=develop)](https://travis-ci.org/BurdaMagazinOrg/thunder-distribution)
+[![Build Status](https://travis-ci.org/BurdaMagazinOrg/thunder-distribution.svg?branch=8.x-2.x)](https://travis-ci.org/BurdaMagazinOrg/thunder-distribution)
 
+Thunder is a Drupal 8 distribution for professional publishers. It consists of the current Drupal 8 functionality, lots of useful modules from publishers and industry partners, and an environment which makes it easy to install, deploy and add new functionality.
 
 
-Thunder is a Drupal 8 distribution for professional publishers. It consists of the current Drupal 8 functionality, lots of useful modules from publishers and industry partners, and an environment which makes it easy to install, deploy and add new functionality.
+## Installing Thunder
+Instructions on how to install Thunder can be found at the [installation guide](https://burdamagazinorg.github.io/thunder-documentation/docs#docs-install) and in the [Drupal documentation](https://www.drupal.org/docs/8/install)..
+
+### Quick start installation
+To start quickly, run this in your terminal to install Thunder from command line:
+
+```
+composer create-project burdamagazinorg/thunder-project thunder --no-interaction
+cd thunder/docroot
+php core/scripts/drupal quick-start thunder
+```
+For further information follow the [install instructions](https://www.drupal.org/docs/8/install/drupal-8-quick-start-command).
+
+If you don't want to use Composer, you can install Thunder the traditional way by downloading a tarball from our [drupal.org](https://www.drupal.org/project/thunder) project page, but we do **not** recommend it.
```

Comment:
This does not belong in this section. But also I think we could remove it as we don't recommend it. Or move it to the bottom if a "If you really have to" section.

Reply:
Removed the sentence about the tarball.